visualization/app/codeCharta/ui/codeMap/codeMap.preRender.service.ts

```diff
@@ -83,21 +80,26 @@ export class CodeMapPreRenderService implements StoreSubscriber, MetricServiceSu
 	}
 
 	private updateRenderMapAndFileMeta() {
-		const unifiedFile: CCFile = this.getSelectedFilesAsUnifiedMap()
+		const unifiedFile = this.getSelectedFilesAsUnifiedMap()
```

Comment:
Why did you remove the type?
```suggestion
		const unifiedFile: CCFile = this.getSelectedFilesAsUnifiedMap()
```

Reply:
Because the type is given by the return value of the function already. Your IDE should recognise that. No need to specify the type here again.